src/operator/operator_tune.cc

```diff
@@ -125,16 +125,16 @@ struct static_init_var {
  * \brief Implement tuning objects for a backward unary kernel operator
  */
 #define _IMPLEMENT_UNARY_WORKLOAD_BWD(__op$, __typ$) \
-  IMPLEMENT_WORKLOAD_VALUE_FOR_TYPE(mxnet::op::mxnet_op::backward_grad<__op$>, __typ$); \
+  IMPLEMENT_WORKLOAD_VALUE_FOR_TYPE(mxnet::op::mxnet_op::backward_grad_tuned<__op$>, __typ$); \
   namespace mxnet_op { \
```

Comment:
don't put namespace in macro.

Reply:
why? this would make the macro sensitive to what namespace it is declared in.

Reply:
Ok, I see what you're saying. The namespaces aren't all the same.